### ISSUE TITLE
[A11Y] Seven theme: Low text contrast issue with inactive primary tabs

### ISSUE THREAD

**Issue by ericvanderwal** [backdrop/backdrop-issues#5043]
**Description of the need**

Text color has insufficient contrast color (compared to the background). Specifically starting line 279 of the CSS block.

See:  **color: #5f5f5f;**

```
ul.primary li a,
ul.primary li a.active,
ul.primary li a:active,
ul.primary li a:visited,
ul.primary li a:hover,
ul.primary li.active a,
.expand-dropdown-tabs-control,
.responsive-tabs-mobile .expand-dropdown-tabs-control:before,
.responsive-tabs-mobile .expand-dropdown-tabs-label {
  display: block;
  padding: 12px 20px;
  border: none;
  overflow: visible;
  line-height: 1;
  color: #5f5f5f;
  font-weight: normal;
  background-color: #DBDBD9;
  transition:
    background-color 0.5s ease 0s,
    color 0.5s ease 0s;
  will-change: background-color, color;
}
```

Text to background should have a minimum of 4.5 to 1 ratio of contrast. Currently the ratio is 3.32 to 1.

You can read more about text contrast ratio here: https://web.dev/color-and-contrast-accessibility/?utm_source=devtools

> The WebAIM guidelines recommend an AA (minimum) contrast ratio of 4.5:1 for all text. An exception is made for very large text (120-150% larger than the default body text), for which the ratio can go down to 3:1. Notice the difference in the contrast ratios shown below.

Official guidelines here: https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html

**Proposed solution**

Purposed solution is to change the color (text color) attribute to be approx. #616060 or darker (this is 4.51 to 1). The change is fairly subtle and does not take away from the general design elements of the theme.

See the example below. Specifically the words "Settings" on the top tab.

***Before Change***

![settings-before](https://user-images.githubusercontent.com/25902459/115061013-26dec880-9f1b-11eb-848b-c527be193524.png)

***After Change***

![settings-after](https://user-images.githubusercontent.com/25902459/115061043-352ce480-9f1b-11eb-8d5d-2747aa62c71f.png)

**Comment by ericvanderwal** [comment]
Open a new ticket for each contrast issue? There are a few.

**Comment by klonos** [comment]
Thanks for opening this issue @ericvanderwal 🙏🏼 ...I agree that this should be fixed, and that the change is subtle 👍🏼 

> Open a new ticket for each contrast issue?

I think yes, unless some of them can be grouped together.

**Comment by klonos** [comment]
@jenlampton and @stpaultim I propose to start using `[A11Y]` for accessibility issues, and add it to the "Issue titles" section at https://backdropcms.org/core-issue-queue

**Comment by ericvanderwal** [comment]
Are we interested in issues effect both Seven and Basic or one in specific?

**Comment by indigoxela** [comment]
A [PR](https://github.com/backdrop/backdrop/pull/3595/files) is available for testing and review. The change is indeed subtle.

As far as I can tell, the same color/background combination isn't used anywhere else (recursive grep).

@klonos Was possible grouping the reason for the "needs feedback" label? I think in this case there's no real opportunity for it, but it might be possible in other cases.

**Comment by ericvanderwal** [comment]
PR reviewed. Now passing WCAG 2.1 tests.

**Comment by klonos** [comment]
@ericvanderwal both Seven and Basis are core themes, so they are supported for the 1.x lifecycle of Backdrop. So yes please, issues for both/either theme.

**Comment by quicksketch** [comment]
Looks great to me. Thanks @ericvanderwal for filing this issue! I'm sure there are many similar contrast problems and I'd love to see them addressed in the same way as this one. I merged https://github.com/backdrop/backdrop/pull/3595 into 1.x and 1.18.x. Thanks @indigoxela, @ericvanderwal, and @klonos!
